contracts/VerifierRegistry.sol

```diff
@@ -73,25 +82,26 @@ contract VerifierRegistry is IVerifierRegistry, Ownable, RegistrableWithSingleSt
     verifier.id = msg.sender;
     verifier.name = _name;
     verifier.location = _location;
-    verifier.active = true;
     verifier.shard = uint256(vrStorage.addressesLength()) / vrStorage.verifiersPerShard();
 
     vrStorage.setVerifier(
+      verifier.active,
```

Comment:
I don't think doing this in alphabetic order makes sense, please leave at same format as before grouping together where they make sense. (e.g. make verifier.id first.)

Reply:
@enddynayn asked me to return verifier values in alphabetic order, so I thought it will be good to have same order everywhere and not to have two different approaches. Taking Enddys request into consideration, should I leave setter as it was @therevoltingx ?

Reply:
Yeah leave it as it was. Alphabetic order doesnt make sense.

Reply:
sounds good to me. The ordering of these variables does not matter, what matters is that we are consistent about where we add new variables. Moving forward lets make sure we add variables to the end so that we don't break things for the client.